01.elements/mediapro/02.advanced/01.customizations/docs.md

```diff
@@ -5,3 +5,5 @@ taxonomy:
 ---
 
 [plugin:insert-content](/_partials/elements/customizations?mediapro)
+
+The global override path for MediaPro is `media/zoo/custom_elements/mediapro/tmpl/render/{player}/myCustom.php` because of players folders inside `/render` folder.
```

Comment:
Make it as a side note, check other docs how is that done and change to text to be like "Because of the players independent folders the MediaPro override paths are...".